### PR TITLE
Third party details on contact page

### PR DESCRIPTION
**Pull request by tyom**

### CODE REVIEW

**Comment by tyom** [comment]
retest this please

File: cla_public/apps/contact/views.py [hunk 1]
```diff
@@ -24,7 +23,7 @@ def add_no_cache_headers(response):
 
 def confirmation_email(data):
     data['case_ref'] = session.checker['case_ref']
-    data['callback_requested'] = data['callback_requested'] == YES
+    data['callback_requested'] = data['contact_type'] in ['callback', 'thirdparty']
     if data['callback_requested']:
         data['safe_to_contact'] = data['callback']['safe_to_contact'] == 'SAFE'
```
Comment (ushkarev) [inline cla_public/apps/contact/views.py]: 'safe to contact' needs to look at third party form instead if somebody else is getting the call back

File: cla_public/templates/checker/result/confirmation.html [hunk 2]
```diff
@@ -41,9 +43,10 @@
 
   {% if session.stored.callback_requested %}
     <p>
-      {% trans %}Your details have been submitted and an operator will try to call
-        you back around the time you requested {% endtrans %}
-        (<strong>{{ session.stored.callback_time|datetime('full') }}</strong>).
+      {{ _('Your details have been submitted and an operator will try to call') }}
```
Comment (ushkarev) [inline cla_public/templates/checker/result/confirmation.html]: probably not a great idea to split a sentence into separate translated strings – the sentence order may not work correctly in welsh and the lack of context might lead to incorrect translation of the fragments. (applies to a few places in this file; but template translations need a review throughout the site anyway)

**Comment by ushkarev** [comment]
screen shot 2015-06-02 at 11 41 55
should the description copy change who it addresses in third-party callback mode?

**Comment by tyom** [comment]
> should the description copy change who it addresses in third-party callback mode?

Updated.

**Comment by tyom** [comment]
test this please

**Pull request merged**